crates/angstrom-net/src/manager.rs

```diff
@@ -64,6 +95,38 @@ impl<DB: Unpin> StromNetworkManager<DB> {
         }
     }
 
+    pub fn load_known_peers() -> Vec<CachedPeer> {
+        CACHED_PEERS_TOML_PATH.with(|toml_path| {
+            match std::fs::read_to_string(toml_path.as_path()) {
+                Ok(data) => toml::from_str(&data).unwrap_or_default(),
+                Err(_) => {
+                    tracing::warn!(
+                        "Known peers file not found at {}, creating peers list from scratch",
+                        toml_path.display()
+                    );
+                    Vec::new()
+                }
+            }
+        })
+    }
+
+    pub fn save_known_peers(peers: &[CachedPeer]) {
+        CACHED_PEERS_TOML_PATH.with(|toml_path| match toml::to_string(peers) {
+            Ok(serialized) => {
+                if let Err(err) = std::fs::write(toml_path.as_path(), serialized) {
+                    tracing::error!(
+                        "Failed to save known peers to {}: {}",
+                        toml_path.display(),
+                        err
+                    );
+                }
+            }
+            Err(err) => {
+                tracing::error!("Failed to serialize known peers to TOML: {}", err);
+            }
+        });
+    }
```

Comment:
will this create the path / file if it doesn't exist? im not 100% sure about what write does